ArchiSteamFarm/Steam/Integration/SteamChatMessage.cs

```diff
@@ -33,6 +33,7 @@ namespace ArchiSteamFarm.Steam.Integration {
 	internal static class SteamChatMessage {
 		internal const char ContinuationCharacter = '…'; // A character used for indicating that the next newline part is a continuation of the previous line
 		internal const byte ContinuationCharacterBytes = 3; // The continuation character specified above uses 3 bytes in UTF-8
+		internal const char ParagraphCharacter = '¶'; // A character used for indicating that this is not the last part of message (2 bytes, so it fits in ContinuationCharacterBytes)
```

Comment:
Since we have a test case for checking `ContinuationCharacter` against `ContinuationCharacterBytes` , I'd like to see one verifying the comment on this line as well.

```cs
[TestMethod]
public void ParagraphCharacterFitsInContinuationCharacterBytes() => Assert.IsTrue(Encoding.UTF8.GetByteCount(ParagraphCharacter.ToString()) <= ContinuationCharacterBytes);
```

Reply:
Makes sense, will do.